### ISSUE TITLE
V1 support

### ISSUE THREAD

**Issue by ataitler** [pyrddlgym-project/pyRDDLGym-prost#2]
Branch for interaction with pyRDDLGym 1.x

**Comment by GMMDMDIDEMS** [comment]
I would recommend providing two stable Docker images (for pyRDDLGym v1 and v2) via Docker Hub. This guarantees that the images will work and saves users the time-consuming build process.

For this we have to keep in mind:
- pyRDDLGym only produces correct Prost logs from version  >= 1.4.3. See https://github.com/pyrddlgym-project/pyRDDLGym/pull/240. Additionally https://github.com/pyrddlgym-project/pyRDDLGym/issues/248 must be fixed for pyRDDLGym 1.x
- Is the issue https://github.com/pyrddlgym-project/pyRDDLGym/issues/238 for any pyRDDLGym 1.x fixed?
- pyRDDLGym 2.0 produced an [IndexError](https://github.com/pyrddlgym-project/pyRDDLGym/issues/248). Therefore version > 2.0 will be stable.

It would probably be best to add this information as a `warning` to prevent users from building Docker images with pyRDDLGym versions that have known issues.

**Comment by mike-gimelfarb** [comment]
Hi,

Thanks for your feedback about it. I agree with most points, especially about hosting a docker image on Hub. 

The issues mentioned are fixed in 2.0, but not yet in 1.4. The https://github.com/pyrddlgym-project/pyRDDLGym/issues/238 is only partially fixed, since there may still be boundary cases that are not handled in the fix (but it should work for all of the typical cases and existing domains that exist). I think it's mainly a matter of backporting all these fixes into 1.4.4. 

We need to decide whether its worth doing all this and changing 1.4.4 in order to support 1.x in the Prost. Functionally, I do not see any point myself because the user does not interact with the pyRDDLGym server in any way with the current setup when running the docker images. For consistency, there is also no longer support for 1.4 and all of the other baselines, and likely will not be.

**Comment by GMMDMDIDEMS** [comment]
I agree, that supporting pyRDDLGym 1.x wouldn't bring any benefits as the (Prost) results will be the same.

With
> pyRDDLGym 2.0 produced an https://github.com/pyrddlgym-project/pyRDDLGym/issues/248. Therefore version > 2.0 will be stable.

I meant the released package. But since the Dockerfile always pulls the latest pyRDDLGym version from Github, you're right, the problem is fixed.
